paymentsheet/src/main/java/com/stripe/android/common/exception/ExceptionKt.kt

```diff
@@ -19,3 +21,17 @@ internal fun Throwable?.stripeErrorMessage(context: Context): String {
     }
     return context.getString(R.string.stripe_something_went_wrong)
 }
+
+@Suppress("ReturnCount")
+internal fun Throwable?.stripeErrorMessage(): ResolvableString {
```

Comment:
Can we mark the other one as deprecated to discourage use?

Reply:
Also, I don’t generally like the idea of an extension on a nullable type. Do you have any strong opinion or are you fine changing this to `Throwable.stripeErrorMessage()`?

Reply:
In looking back at this, there is a use case where having this on a nullable `Throwable` allows for resolving error messages on failures that don't produce a `Throwable` for some reason. Do we want to maintain this behavior or force developers to always ensure a `Throwable` is returned as part of their failure flow?

Reply:
I’d move this up to the caller of `stripeErrorMessage()`. It should decide the error message if the `cause` is null.